main/sdr.go

```diff
@@ -147,12 +149,14 @@ func getPPM(serial string) int {
 
 func (e *ES) sdrConfig() (err error) {
```

Comment:
Why is this function so empty compared to the func (u *UAT) sdrConfig() (err error)